app.rb

```diff
@@ -24,6 +25,14 @@ def select_option(user_response)
       create_rental
     when 6
       rental_by_id
+    when 7
```

Comment:
@Metaverse-Nabeel, currently we are storing the data only in the json file, but whet we close the session of the program (option 7) all the information is deleted and in the next time that we initialize the program, all the json information is gonna be overwritten. We need to find a way to preserve the data to avoid that problem. I was thinking in something like add local storage.